PizzaPyWebApp/templates/partials/about_us/history.html

```diff
@@ -0,0 +1,95 @@
+<style>
+  body {
+    scroll-behavior: smooth;
+  }
+
+  .timeline-item {
+    opacity: 0;
+    transform: translateY(50px);
+    transition: all 0.5s ease-out;
+  }
+
+  .timeline-item.visible {
+    opacity: 1;
+    transform: translateY(0);
+  }
+</style>
+
+<div class="flex flex-col items-center gap-10 p-6 sm:gap-20 md:p-28">
+  <div class="flex flex-col items-center w-full">
+    <p class="text-4xl font-extrabold text-center text-base-content sm:text-6xl md:text-7xl">
+      How did PizzaPy come to be?
+    </p>
+    <p class="text-4xl font-extrabold text-center text-yellow-500 sm:text-6xl md:text-7xl">
+      here’s some history.
+    </p>
+  </div>
+</div>
+
+<div class="flex justify-center p-4">
+  <div id="timeline"
+    class="space-y-8 relative before:absolute before:inset-0 before:ml-5 before:-translate-x-px md:before:mx-auto md:before:translate-x-0 before:h-full before:w-0.5 before:bg-gradient-to-b before:from-transparent before:via-slate-300 before:to-transparent">
+    <!-- Timeline items will be appended here -->
+  </div>
+</div>
+
+<div class="text-center text-[56px] font-extrabold text-white">And further beyond...</div>
+
+<script>
+  document.addEventListener('DOMContentLoaded', () => {
+    fetch('../../../static/data/history.json')
+      .then(response => response.json())
+      .then(timelineData => {
+        const timelineContainer = document.getElementById('timeline');
+
+        timelineData.forEach((item, index) => {
+          const isActiveClass = index === 0 ? 'group is-active' : 'group';
+          const dateColor = item.title === 'Delivered' ? 'text-amber-500' : 'text-indigo-500';
+          const isOdd = index % 2 !== 0;
+
+          const timelineItem = `
+              <div class="timeline-item relative flex items-center justify-between md:justify-normal md:odd:flex-row-reverse ${isActiveClass}">
+                <!-- Icon -->
+                <div class="flex items-center justify-center w-10 h-10 rounded-full border border-white bg-[#243A50] group-[.is-active]:bg-emerald-500 text-white group-[.is-active]:text-emerald-50 shadow shrink-0 md:order-1 md:group-odd:-translate-x-1/2 md:group-even:translate-x-1/2"></div>
+                <!-- Card -->
+                <div class="w-[calc(100%-4rem)] flex md:w-[calc(50%-2.5rem)] bg-[#0C1622] p-4 rounded border border-[#2B3A50] shadow">
```

Comment:
Please lessen the padding on mobile screens.

The inner items have a `p-4` and the card itself has `p-4`, making the card contents too compressed in mobile views. You can use conditional TailwindCSS selectors for this.

See my video below for an example:
![demo](https://github.com/pizzapy/pizzapy-website/assets/105530193/a5bb75b1-cbfe-4a22-aebd-abcf7ddde1b4)